### PR TITLE
Fix Gagandeep's name

### PR DESCRIPTION
**Pull request by certik**
@czgdp1807 do you want your name like this?
```
  8400  Ondřej Čertík
  1662  ਗਗਨਦੀਪ ਸਿੰਘ (Gagandeep Singh)
  1012  Ubaid Shaikh
   798  Thirumalai Shaktivel
   534  Pranav Goswami
   284  Smit Lunagariya
   100  Rohit Goswami
    92  Luthfan Anshar Lubis
    91  Dominic Poerio
    86  Naman Gera
    64  Harshita Kalani
    51  Sarthak Gupta
    47  Andrew Best
    37  Tapasweni Pathak
    35  Carlos Une
    24  Harris M. Snyder
    22  Zihua Wu
    17  Jacob Marks
    15  Konrad Handrick
    15  Himanshu Pandey
    13  Khushi Agrawal
    11  左志华 (Zuo Zhihua)
    11  Sebastian Ehlert
    11  Giannis Nikiteas
    10  meow464
     9  Oshanath Rajawasam
...
```

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # The "Name <email>" on the left is the desired name and email
-Gagandeep Singh <gdp.1807@gmail.com> czgdp1807 <gdp.1807@gmail.com>
-Gagandeep Singh <gdp.1807@gmail.com> ਗਗਨਦੀਪ ਸਿੰਘ (Gagandeep Singh) <gdp.1807@gmail.com>
-Gagandeep Singh <gdp.1807@gmail.com> ਗਗਨਦੀਪ ਸਿੰਘ <gdp.1807@gmail.com>
+ਗਗਨਦੀਪ ਸਿੰਘ (Gagandeep Singh) <gdp.1807@gmail.com> czgdp1807 <gdp.1807@gmail.com>
+ਗਗਨਦੀਪ ਸਿੰਘ (Gagandeep Singh) <gdp.1807@gmail.com> ਗਗਨਦੀਪ ਸਿੰਘ <gdp.1807@gmail.com>
+ਗਗਨਦੀਪ ਸਿੰਘ (Gagandeep Singh) <gdp.1807@gmail.com> Gagandeep Singh <gdp.1807@gmail.com>
```
Comment (czgdp1807) [inline .mailmap:4]:
```suggestion
Gagandeep Singh <gdp.1807@gmail.com> czgdp1807 <gdp.1807@gmail.com>
```

**Review by czgdp1807 — CHANGES_REQUESTED** [review]

File: .mailmap (hunk 1 above)
Comment (Shaikh-Ubaid) [reply .mailmap]: With the above change, it seems there are total three contributors in the output:

```console
$ git shortlog -nse
...
    1629  Gagandeep Singh <gdp.1807@gmail.com>
    32  ਗਗਨਦੀਪ ਸਿੰਘ (Gagandeep Singh) <gdp.1807@gmail.com>
     1  ਗਗਨਦੀਪ ਸਿੰਘ <gdp.1807@gmail.com>
(END)
```

@czgdp1807 Is the above output fine?

Comment (Thirumalai-Shaktivel) [reply .mailmap]: Nope, @czgdp1807 we have to map all the occurrences of your account ID (name) to one specific name.
The previous change was correct.

@certik asked for whether to keep your name as `ਗਗਨਦੀਪ ਸਿੰਘ (Gagandeep Singh)` or `Gagandeep Singh`

Comment (certik) [reply .mailmap:4]: @czgdp1807 waiting for your decision ^^^.

Comment (czgdp1807) [reply .mailmap:4]: Keep second one i.e., english only one.

Comment (certik) [reply .mailmap:4]: Thanks, that's already implemented in master:
```
  8400  Ondřej Čertík
  1662  Gagandeep Singh
  1012  Ubaid Shaikh
   798  Thirumalai Shaktivel
   536  Pranav Goswami
   284  Smit Lunagariya
   100  Rohit Goswami
    92  Luthfan Anshar Lubis
    91  Dominic Poerio
    86  Naman Gera
    64  Harshita Kalani
    51  Sarthak Gupta
    47  Andrew Best
    37  Tapasweni Pathak
    35  Carlos Une
    24  Harris M. Snyder
    22  Zihua Wu
    17  Jacob Marks
    15  Konrad Handrick
    15  Himanshu Pandey
    13  Khushi Agrawal
    11  左志华 (Zuo Zhihua)
    11  Giannis Nikiteas
    11  Sebastian Ehlert
    10  meow464
```

**Pull request closed without merging**